fastlane/Fastfile

```diff
@@ -570,15 +592,23 @@ REPOSITORY_NAME="WordPress-Android"
     }
 
     delete_old_changelogs(build: options[:build_number])
-    download_path=Dir.pwd + "/metadata/android"
-    gp_downloadmetadata(project_url: "https://translate.wordpress.org/projects/apps/android/release-notes/",
+    download_path = File.join(Dir.pwd, paths[:metadata_dir], 'android')
+    gp_downloadmetadata(
+      project_url: paths[:gp_url],
       target_files: files,
-      locales: RELEASE_NOTES_LOCALES,
+      locales: RELEASE_NOTES_LOCALES, # Currently the same for WP and JP
```

Comment:
Well after running the lane I realized that's quite not true, as it seems JP will have less locales than JP for release notes. I'll address this later next week.